src/main/kotlin/com/evalify/evalifybackend/semester/domain/DTO/SemesterManagerDTO.kt

```
@@ -0,0 +1,8 @@
+package com.evalify.evalifybackend.semester.domain.DTO
+
+class SemesterManagerDTO(
+    val name: String,
+    val id: String?,
+    val profileId: String?
+) {
+}
```

detekt (1.23.8)

[warning] The class or object SemesterManagerDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)